components/style/str.rs

```diff
@@ -14,29 +14,11 @@ use std::fmt::{self, Write};
 use std::iter::{Filter, Peekable};
 use std::str::Split;
 
-/// A static slice of characters.
-pub type StaticCharVec = &'static [char];
+pub use style_traits::{char_is_whitespace, HTML_SPACE_CHARACTERS, StaticCharVec};
```

Comment:
Can we change the consumers of this instead of reexporting? Also I suspect there are a couple of them that don't even need to be public.